### PR TITLE
[select] select doc update v11

### PR DESCRIPTION
**Pull request by thyhmdo**
Original issue: https://github.com/carbon-design-system/carbon-website/issues/3241
Updated content and images for the [Select](https://carbondesignsystem.com/components/select/usage/) component.

### CODE REVIEW

**Review by alisonjoseph — APPROVED** [review]

File: src/pages/components/select/usage.mdx [hunk 1]
```diff
@@ -18,30 +17,72 @@ in forms for users to submit data.
...
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Behaviors</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
 ## Overview
 
-Select is a type of input that is used in forms, where a user is submitting data
+The select component is used for collecting user-provided information from a
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:29]:
```suggestion
The select component collects user-provided information from a
```

File: src/pages/components/select/usage.mdx [hunk 2]
```diff
@@ -18,30 +17,72 @@ in forms for users to submit data.
...
+list of options. Selects are usually used in forms, where a user submits data
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:30]:
```suggestion
list of options. Selects are usually used in forms where a user submits data
```

File: src/pages/components/select/usage.mdx [hunk 3]
```diff
@@ -18,30 +17,72 @@ in forms for users to submit data.
...
+It is best practice not to use a select if there are fewer than three options
+for selection. In this case, use a
+[radio button](https://carbondesignsystem.com/components/radio-button/usage/)
+group instead.
+
+#### Select versus Dropdown
+
+While the select and
+[dropdown](https://carbondesignsystem.com/components/dropdown/usage/) components
+look similar, but they have different functions.
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:59]:
```suggestion
look similar, they have different functions.
```

File: src/pages/components/select/usage.mdx [hunk 4]
```diff
@@ -18,30 +17,72 @@ in forms for users to submit data.
...
+
+- A select presents a list of options from which the users can select only one
+  item from that list. It works best in forms when users select an option from
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:62]:
```suggestion
  It works best in forms when users choose an option from
```

File: src/pages/components/select/usage.mdx [hunk 5]
```diff
@@ -18,30 +17,72 @@ in forms for users to submit data.
...
+The select component’s appearance will be determined by the browser being used,
 while the dropdown component can be styled as needed.
 
+<Row>
+<Column colLg={12}>
+
+![Dropdown versus Select](images/select-usage-2.png)
+
+<Caption>
+  The example shows that the dropdown presents a list of options for actions
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:79]:
```suggestion
  Example of a dropdown list of actional options
```

File: src/pages/components/select/usage.mdx [hunk 6]
```diff
@@ -141,59 +178,186 @@ Inline selects have less visual weight on a page because they are borderless.
...
+
+### States
+
+The select has multiple interactive states for both default and inline variants:
+**enabled**, **hover**, **focus**, **open**, **error**, **warning**, and
+**disabled**.
+
+| State       | When to use                                                                                                                                                                                                                                                  |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| _Enabled_   | When the select is live but a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component. An enabled select field should contain a default value.                                                |
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:275]:
```suggestion
| _Enabled_   | When the select is live and a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component. An enabled select field should contain a default value.                                                |
```

File: src/pages/components/select/usage.mdx [hunk 7]
```diff
@@ -141,59 +178,186 @@ Inline selects have less visual weight on a page because they are borderless.
...
+| _Error_     | When the required select value has not been selected. It can also be triggered due to a system error. This state requires a user response before data can be submitted or saved.                                                                             |
+| _Warning_   | When you need to call the user's attention to an exception condition. The condition might not be an error but can cause problems if not resolved.                                                                                                            |
+| _Disabled_  | When the user is not allowed to interact with the select due to either permissions, dependencies, or prerequisites. The disabled state completely removes the interactive function from a component. The styling is not subject to WCAG contrast compliance. |
+| _Skeleton_  | Used on an initial page load to indicate that the number input has not yet fully loaded.                                                                                                                                                                     |
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:281]:
```suggestion
| _Skeleton_  | Used on an initial page load to indicate that the select has not yet fully loaded.                                                                                                                                                                     |
```

File: src/pages/components/select/usage.mdx [hunk 8]
```diff
@@ -141,59 +178,186 @@ Inline selects have less visual weight on a page because they are borderless.
...
+| _Read-only_ | _Coming soon!_                                                                                                                                                                                                                                               |
+
+<Row>
+<Column colLg={12}>
+
+![Select interactive states](images/select-usage-9.png)
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:287]: — I think you could remove the word `state` in the labels of each component in the image.
— The disabled state does not get a bottom-border rule.

File: src/pages/components/select/usage.mdx [hunk 9]
```diff
@@ -141,59 +178,186 @@ Inline selects have less visual weight on a page because they are borderless.
...
+- Users can move the highlighted option to the previous option by pressing the
+  `Up arrow`.
+- Users can close the dropdown menu by pressing `Escape`, `Space`, or `Enter`.
 
-Labels are essential to the usability of forms. Do not place a label inside a
-_select_ element. Use sentence case and no more than three words.
+<Row>
+<Column colLg={8}>
+
+![Select keyboard interaction](images/select-usage-11.png)
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:324]: I notice in the second part of this image you have the pink overlay on the field input, I don't think it's needed in this image.

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looking great Thy, sorry for my late review! Most of my review consists of some small content typos.

**Review by tw15egan — APPROVED** [review]

File: src/pages/components/select/usage.mdx [hunk 10]
```diff
@@ -18,30 +17,75 @@ in forms for users to submit data.
...
+- When the experience is mostly form-based.
+
+#### When not to use
+
+It is best practice not to use a select if there are fewer than three options
+for selection. In this case, use a
+[radio button](https://carbondesignsystem.com/components/radio-button/usage/)
+group instead.
+
+#### Select versus Dropdown
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:55]: There are some weird code things going on here in the Select versus Dropdown section.
Screen Shot 2022-12-12 at 10 48 37 AM

File: src/pages/components/select/usage.mdx [hunk 11]
```diff
@@ -141,59 +181,187 @@ Inline selects have less visual weight on a page because they are borderless.
...
 <Column colLg={8}>
 
-![Example of a small select](images/select-usage-4.png)
+![Example of a inline select](images/select-usage-4.png)
 
 </Column>
 </Row>
 
-## Best practices
+## Formating
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:189]:
```suggestion
## Formatting
```

File: src/pages/components/select/usage.mdx [hunk 12]
```diff
@@ -141,59 +181,187 @@ Inline selects have less visual weight on a page because they are borderless.
...
+
+<Row>
+<Column colLg={8}>
+
+![Select mouse interaction](images/select-usage-10.png)
+
+</Column>
+</Row>
+
+#### Keyboard
```
Comment (laurenmrice) [inline src/pages/components/select/usage.mdx:312]: There is another code bug happening here.
Screen Shot 2022-12-12 at 10 52 53 AM

**Review by laurenmrice — CHANGES_REQUESTED** [review]
@thyhmdo A couple small things Thy! Just saw some code errors happening in the content and a typo. Otherwise looks good!

**Review by laurenmrice — APPROVED** [review]
Looks great Thy! ⭐️

**Pull request merged**